netadapter/server/grpcserver/protowire/messages.proto

```diff
@@ -3,11 +3,188 @@ package protowire;
 
 option go_package = "github.com/kaspanet/kaspad/protowire";
 
-message KaspadMessage{
-  uint32 command = 1;
-  bytes payload = 2;
+message KaspadMessage {
+  oneof payload {
+    AddressesMessage addresses = 1;
+    BlockMessage block = 2;
+    TransactionMessage transaction = 3;
+    BlockLocatorMessage blockLocator = 4;
+    GetAddressesMessage getAddresses = 5;
+    GetBlocksMessage getBlocks = 6;
+    GetRelayBlocksMessage getRelayBlocks = 7;
+    GetSelectedTipMessage getSelectedTip = 8;
+    GetTransactionsMessage getTransactions = 9;
+    BlockMessage ibdBlock = 10;
+    InvRelayBlockMessage invRelayBlock = 11;
+    InvTransactionsMessage invTransactions = 12;
+    PingMessage ping = 13;
+    PongMessage pong = 14;
+    SelectedTipMessage selectedTip = 15;
+    VerackMessage verack = 16;
+    VersionMessage version = 17;
+  }
 }
 
+// AddressesMessage Start
+message AddressesMessage{
+  bool includeAllSubnetworks = 1;
+  SubnetworkID subnetworkID = 2;
+  repeated NetAddress addressList = 3;
+}
+
+message NetAddress{
+  int64  timestamp = 1;
+  uint64 services = 2;
+  bytes ip = 3;
+  uint32 port = 4;
+}
+
+message SubnetworkID{
+  bytes bytes = 1;
+}
+// AddressesMessage End
+
+// GetAddressesMessage start
+message GetAddressesMessage{
+  bool includeAllSubnetworks = 1;
+  SubnetworkID subnetworkID = 2;
+}
+// GetAddressesMessage end
```

Comment:
Sometimes the start/end are Upper Case, sometimes they are lower case. Please be consistent